### PR TITLE
Refactor metric and alert names 

### PR DESCRIPTION
**Pull request by dimitraz**
## Overview

Jira: https://issues.redhat.com/browse/INTLY-5109

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Create a managed postgres instance and a managed redis instance:
```
make cluster/seed/managed/redis
make cluster/seed/managed/postgres
```
- Ensure the metrics `cro_redis_available` and `cro_postgres_available` are being exposed
- Ensure that the promethus rules are getting created and 
- Ensure that alert names, descriptions, etc do not contain provider specific terminology (eg should be Postgres over RDS)

### CODE REVIEW

File: pkg/providers/aws/provider_postgres.go [hunk 1]
```diff
@@ -35,14 +33,13 @@ import (
...
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	defaultPostgresMaintenanceMetricName = "cro_aws_rds_service_maintenance"
-	defaultPostgresInfoMetricName        = "cro_aws_rds_info"
-	defaultPostgresAvailMetricName       = "cro_aws_rds_available"
+	defaultPostgresMaintenanceMetricName = "cro_postgres_service_maintenance"
+	defaultPostgresInfoMetricName        = "cro_postgres_info"
+	defaultPostgresAvailMetricName       = "cro_postgres_available"
```
Comment (david-martin) [inline pkg/providers/aws/provider_postgres.go]: @ciaranRoche Some changes to metrics names to be aware of

**Comment by davidkirwan** [comment]
```
cro_postgres_available{clusterID="dimitra-brxhg",endpoint="http-metrics",exported_namespace="redhat-rhmi-operator",instance="10.128.2.23:8383",instanceID="dimitrabrxhgredhatrhmioperatorupspo-o2nx",job="cloud-resource-operator-metrics",namespace="redhat-rhmi-cloud-resources-operator",pod="cloud-resource-operator-686c947648-p2z7j",productName="ups",resourceID="ups-postgres-integreatly",service="cloud-resource-operator-metrics",strategy="aws-rds"} | 1
-- | --
```

- Manually deleted an RDS instance
- metric now returning `0` for unavailable

```
cro_postgres_available{clusterID="dimitra-brxhg",endpoint="http-metrics",exported_namespace="redhat-rhmi-operator",instance="10.128.2.23:8383",instanceID="dimitrabrxhgredhatrhmioperatorupspo-o2nx",job="cloud-resource-operator-metrics",namespace="redhat-rhmi-cloud-resources-operator",pod="cloud-resource-operator-686c947648-p2z7j",productName="ups",resourceID="ups-postgres-integreatly",service="cloud-resource-operator-metrics",strategy="aws-rds"} | 0
-- | --
```

**Pull request merged**